lib/tasks/transcribe.js

```diff
@@ -99,6 +99,15 @@ class TaskTranscribe extends SttTask {
       this.data.recognizer.model = cs.speechRecognizerLanguage;
     }
 
+    if (cs.recognizer) {
```

Comment:
since we are doing this in two places, is there an opportunity to factor it into stt-task?